webrtc-stats.html

```diff
@@ -1853,6 +1855,28 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>lastPacketSentTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the timestamp at which the last packet
+                  was sent on this particular candidate pair.
+                  It is of the type <code>DOMHighResTimeStamp</code> [[!HIGHRES-TIME]].
+                </p>
```

Comment:
The last sentence merely repeats the type, which is already stated above. Please remove it.

If you want a ref for DOMHighResTimeStamp, put it in the terminology section.